src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -523,7 +523,7 @@ private String resolveContent(String result, Set<String> usedIds) {
      */
     public static Optional<String> getResolvedField(String field, BibEntry entry, BibDatabase database) {
         Objects.requireNonNull(entry, "entry cannot be null");
-        if ("bibtextype".equals(field)) {
+        if (BibEntry.TYPE_HEADER.equals(field) || "bibtextype".equals(field)) {
```

Comment:
This has got a bit more clear now. At some stage the name of the entry type was changed from `bibtextype` to `entrytype`, but not completely, only in parts of the code, which is evident from the old code and all layout. However, when I tested to use `bibtextype` in a column nothing was shown, not clear why though. Anyway, the changes in #1651 are unified in this PR and maybe we should simply remove "bibtextype", but better wait for a while. No need for any migration as "bibtextype" didn't work in a column anyway and we do not want to modify peoples layout files...